Person/Person/main.m

```diff
@@ -51,13 +60,81 @@ - (NSString *)phoneNumber {
     return _phoneNumber;
 }
 
+- (BOOL)checkSameCity:(Person *)person {
+    if ([[self city] isEqualToString:[person city]]) {
+        return YES;
+    } else {
+        return NO;
+    }
+}
+
+- (BOOL)checkSamePhoneNumber:(Person *)person {
+    if ([[self phoneNumber] isEqualToString:[person phoneNumber]]) {
+        return YES;
+    } else {
+        return NO;
+    }
+}
```

Comment:
This is 100% correct. Could be a bit more concise by simply returning the value of the `isEqualToString` method...

```objective-c
- (BOOL)checkSamePhoneNumber:(Person *)person {
 return  [[self phoneNumber] isEqualToString:[person phoneNumber]];
}
```